### ISSUE TITLE
Release 1.9.7 Java files were compiled with JDK14 

### ISSUE THREAD

**Issue by rbarazzutti** [scalate/scalate#411]
**Describe the bug**
Release 1.9.7 Java files were compiled with JDK14 (class `major version 58`).
Prior to this, release `1.9.6` Java files were compiled with JDK8 (class `major version 52`).

This applies to release `1.9.7` for `Scala211`, `Scala212`, and `Scala213`.

I imagine that most users would be happy with Java byte code of the JDK8.

According to [the JDK compatibility of Scala recommendations page](https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html), _compiling on JDK8 may be the safer choice_.

**To Reproduce**

```
shell> wget https://repo1.maven.org/maven2/org/scalatra/scalate/scalate-core_2.13/1.9.7/scalate-core_2.13-1.9.7.jar
Saving to: ‘scalate-core_2.13-1.9.7.jar’
shell> javap  -verbose  -cp scalate-core_2.13-1.9.7.jar org.fusesource.scalate.japi.Converter | grep major
  major version: 58
shell>
```

**Comment by magnolia-k** [comment]
@seratch 

Could you check the environment for the release build?

**Comment by alex-dyner** [comment]
Is there any way to republish this artifact or build a new minor version compatible with JDK11 or JDK8?

**Comment by magnolia-k** [comment]
@seratch 

Can you publish artifacts built with JDK8?

**Comment by seratch** [comment]
I just released v1.9.8 and the version does not have this issue.
```
$ wget -q https://repo1.maven.org/maven2/org/scalatra/scalate/scalate-core_2.13/1.9.8/scalate-core_2.13-1.9.8.jar
$ javap  -verbose  -cp scalate-core_2.13-1.9.8.jar org.fusesource.scalate.japi.Converter | grep major
  major version: 52
```
